app/components/Exchange/MyOpenOrders.jsx

```diff
@@ -55,13 +56,19 @@ class TableHeader extends React.Component {
                             </span>
                         ) : null}
                     </th>
-                    <th style={leftAlign ? leftAlign : rightAlign}>
+                    <th style={leftAlign ? rightAlign : leftAlign}>
```

Comment:
Switcharooooo

Reply:
Yes, because the expiration should be inversed in alignment based on the choice.